### DISCUSSION TITLE
B matrix definition

### DISCUSSION THREAD

**Discussion by tingkai-li** [cooperrc/advanced-dynamics#52] [Q&A] [answered]
I found an inconsistency in the definition of B matrix. In the problem statement of Quiz 8, the B matrix is defined as 

$$
\mathbf{B} = \begin{bmatrix}  \mathbf{C_{qd}^{-1} C_{qi}}\\\1    \end{bmatrix};
$$

but in the lecture note and textbook, it is defined as

$$
\mathbf{B} = \begin{bmatrix}  -\mathbf{C_{qd}^{-1} C_{qi}}\\\1    \end{bmatrix}.
$$

Which definition should be the correct one to use?

**Comment by travismalave** [comment, marked as answer]
During the Q&A it was agree that the second is correct (-Cqd^-1) and the Quiz 08 had a typo.

**Reply by tingkai-li** [reply]
Thank you! I wasn't able to make it to today's session.
